app/models/merchant.rb

```diff
@@ -1,7 +1,8 @@
 class Merchant < ApplicationRecord
   enum :status, {"enabled" => 0, "disabled" => 1}
 
-  validates_presence_of :name
+  validates_presence_of :name, presence: true
```

Comment:
I think we might have the syntax wrong here. I think we'll want validates :name, presence: true